### PR TITLE
fix: correctly handle body requests that are falsy

### PR DESCRIPTION
**Pull request by dgellow**

### CODE REVIEW

File: packages/http/src/validator/__tests__/index.spec.ts [hunk 1]
```diff
@@ -77,6 +77,111 @@ describe('HttpValidator', () => {
...
+      afterAll(() => {
+        // Re-setup the mocks that were defined in the outer beforeAll
+        jest.spyOn(validators, 'validateQuery').mockReturnValue(E.left([mockError]));
+        jest.spyOn(validators, 'validateBody').mockReturnValue(E.left([mockError]));
+        jest.spyOn(validators, 'validateHeaders').mockReturnValue(E.left([mockError]));
+        jest.spyOn(validators, 'validatePath').mockReturnValue(E.left([mockError]));
+      });
+
+      beforeEach(() => jest.restoreAllMocks());
+
```
Comment (bruce-hill) [inline packages/http/src/validator/__tests__/index.spec.ts]: I'd also add a test to make sure it correctly handles `null` as the request body for `{"type": "null"}` schemas
